qa/rpc-tests/nodehandling.py

```diff
@@ -55,7 +55,7 @@ def run_test(self):
         for node in self.nodes[0].getpeerinfo():
             assert(node['addr'] != url.hostname+":"+str(p2p_port(1)))
 
-        connect_nodes_bi(self.nodes,0,1) #reconnect the node
```

Comment:
Just a double check for making sure everything is fine: we moved from `connect_nodes_bi` to `connect_nodes`, is it fine?

Reply:
Yes, the functionality and limitation of `connect_nodes_bi` changed slightly with this PR